### PR TITLE
Improve overview page

### PR DESCRIPTION
**Pull request by mosabua**
## Description

- Fix grammar
- Resize and improve image
- Add slide deck and restructure Trino Summit presentation info

## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -15,16 +15,18 @@ gateway for multiple [Trino](https://trino.io) clusters.
...
 
-![Trino Gateway - High Level Architecture](assets/TrinoGateway-HighLevelArchitecture.png)
+![High-level architecture](assets/high-level-architecture.png)
 
 ## Other resources
 
 * [Trino Gateway announment blog post](https://trino.io/blog/2023/09/28/trino-gateway)
-* :fontawesome-brands-youtube:{ .youtube } [Many clusters and only one
-  gateway presentation at Trino Summit 2023](https://www.youtube.com/watch?v=2qwBcKmQSn0)
-* [Lyft blog post](https://eng.lyft.com/trino-infrastructure-at-lyft-b10adb9db01)
```
Comment (ebyhr) [inline docs/index.md:27]: Why did you remove this post?

**Review by rdsarvar — APPROVED** [review]
outside of @ebyhr's comments on the Lyft post LG2M

i guess the diagram was too large?

File: docs/index.md (hunk 1 above)
Comment (mosabua) [reply docs/index.md:27]: Good question.. dumb mistake of mine is the answer

Comment (mosabua) [reply docs/index.md:27]: Fixed.

**Review by ebyhr — APPROVED** [review]

**Comment by mosabua** [comment]
Thanks @ebyhr  .. merging from Seattle airport waiting for my connection.
